### PR TITLE
'Divio Cloud' instances replaced by 'Divio'

### PR DESCRIPTION
**Pull request by mebdivio**

### CODE REVIEW

File: how-to/create-addon.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ How to package a Django application as an addon
 Register the addon
 ------------------
 
-Before your addon can be uploaded, the Divio Cloud must be ready to receive it.
+Before your addon can be uploaded, Divio must be ready to receive it.
```
Comment (evildmp) [inline how-to/create-addon.rst:9]: _...the Divio Control Panel must be ready..._

File: how-to/django-migrate-existing-project.rst [hunk 2]
```diff
@@ -228,11 +228,11 @@ You're now in a position to test the local site, which should be done thoroughly
...
 
 
-Upload your changes back to the Divio Cloud
--------------------------------------------
+Upload your changes back to the Divio
+-------------------------------------
 
 Your project is a Git repository (certain files and directories are excluded), and should be
-pushed to the Divio Cloud's Git server in the usual way (``git add``/``git commit``/``git push``).
+pushed to the Divio's Git server in the usual way (``git add``/``git commit``/``git push``).
```
Comment (evildmp) [inline how-to/django-migrate-existing-project.rst:235]: _...to Git..._

File: how-to/create-addon.rst (hunk 1 above)
Comment (mebdivio) [reply how-to/create-addon.rst:9]: Done

File: how-to/django-migrate-existing-project.rst (hunk 2 above)
Comment (mebdivio) [reply how-to/django-migrate-existing-project.rst:235]: Done

**Pull request merged**